docs/100-reference/03-acornfile.md

```diff
@@ -918,12 +911,11 @@ volumes: data: {
 
 ### class
 
-`class` refers to the `storageclass` within kubernetes.
+`class` refers to the storage solution to use.
```

Comment:
We should provide a link to storage offerings. As an end user, that's what you would be able to find and use.